src/EnergyPlus/HVACUnitarySystem.cc

```diff
@@ -3136,6 +3122,10 @@ namespace HVACUnitarySystem {
 				UnitarySystem( UnitarySysNum ).ControlType = LoadBased;
 			} else if ( SameString( Alphas( iControlTypeAlphaNum ), "SetPoint" ) ) {
 				UnitarySystem( UnitarySysNum ).ControlType = SetPointBased;
+			} else if ( SameString( Alphas( iControlTypeAlphaNum ), "ASHRAE90VariableFan" ) ) {
+				UnitarySystem( UnitarySysNum ).ControlType = CCM_ASHRAE;
```

Comment:
Yes, this is the key field from the idd. I'll test to make sure it works and check in a final today.